frontend/public/components/utils/firehose.jsx

```diff
@@ -71,9 +71,20 @@ const worstError = errors => {
   return worst;
 };
 
+const mapStateToProps = ({k8s}) => ({
+  k8s,
+});
+
+const propsAreEqual = (prevProps, nextProps) => {
+  if (nextProps.children === prevProps.children && nextProps.reduxes === prevProps.reduxes) {
```

Comment:
There are several other props missing from this check: `filters, loaded, loadError, resources`

Reply:
how so ? `ConnectToState` has only 3 props - `children`, `reduxes` (both from Firehose) and `k8s` (from mapStateToProps). Am I wrong ?

Reply:
Sorry you're right. Not sure how I came to that conclusion before.